modules/ROOT/partials/configuration/a11y_advanced_options.adoc

```diff
@@ -3,12 +3,14 @@
 
 This option affects the functionality of:
 
-* The Accessibility Checker plugin (`+a11ychecker+`).
-* The Image plugin (`+image+`).
+* The xref:a11ychecker.adoc[Accessibility Checker] plugin (`+a11ychecker+`).
+* The xref:image.adoc[Image] plugin (`+image+`).
+* The xref:uploadcare.adoc[Image Optimizer Powered by Uploadcare] plugin (`+uploadcare+`).
 
 Setting `+a11y_advanced_options+` to `+true+`:
```

Comment:
@kemister85 Ensure "Decorative Image" icon is covered in "Image Operations" section. 
Note: "Decorative Image" icon only appears when "a11y_advanced_options: true,"
<img width="553" alt="image" src="https://github.com/user-attachments/assets/6dd09eb5-67a7-486e-bd07-48ce79bcf172">

Reply:
Added and included a note that the `a11y_advanced_option` must be set for this to show in the context menu.